pkg/storaged/stratis-panel.jsx

```diff
@@ -119,10 +120,15 @@ export function create_stratis_pool(client) {
                           value: name,
                           validate: name => validate_pool_name(client, null, name)
                       }),
-            CheckBoxes("encrypt", "",
+            CheckBoxes("encrypt_pass", "",
                        {
                            fields: [
-                               { tag: "on", title: _("Encrypt data") }
+                               {
+                                   tag: "on",
+                                   title: (client.features.stratis_crypto_binding
+                                       ? _("Encrypt data with a passphrase")
+                                       : _("Encrypt data"))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230602-113720-366c6399-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)